libs/libarchopenfpga/src/bitstream_setting.h

```diff
@@ -162,6 +171,7 @@ class BitstreamSetting {
   vtr::vector<BitstreamInterconnectSettingId, std::string>
     interconnect_default_paths_;
   std::vector<NonFabricBitstreamSetting> non_fabric_;
+  std::vector<PathBitSetting> path_bit_settings_;
```

Comment:
In my view, there is no need to create a new struct here. Two vectors can work well:

```
std::vector<std::string> overwrite_bit_paths_;
std::vector<std::string> overwrite_bit_values_;
```

Reply:
I do not agree, I can see the code is extremely not efficient, take an example on interconnect bitstream settings. There are 4 elements of data we need to store, we create 4 + 1 vector for each element, and create 4 function to access each element - https://github.com/lnis-uofu/OpenFPGA/blob/master/libs/libarchopenfpga/src/bitstream_setting.h#L155-L163, https://github.com/lnis-uofu/OpenFPGA/blob/master/libs/libarchopenfpga/src/bitstream_setting.h#L92-L99 which can be simplified with a struct.

Yeah for now overwrite_bitstream only need two elements, it does not seems a lot, but still it complicate the code (in my point of view) and who know in the future if we want to expand it or not.

Although I do not agree, I still made the change via https://github.com/chungshien/OpenFPGA/commit/2a3d69aded840c54212dd9652670ab54e9ab1222